tools/validators/ontology_validator/yamlformat/validator.py

```diff
@@ -22,7 +22,7 @@
 from os import path
 import sys
 
-from arg_parser import CreateParser
+from google3.third_party.digitalbuildings.tools.validators.ontology_validator.yamlformat.arg_parser import CreateParser
```

Comment:
this is not correct, update the copybara mapping rules?

Reply:
Will update copy.bara.sky to fix this and send a CL.